### ISSUE TITLE
Hide/Filter Offline Node/User

### ISSUE THREAD

**Issue by IamTaoChen** [GoodiesHQ/headscale-admin#30]
Maybe we can filter the nodes and users, e.g. Online/Offline.

And for routes, we should hide the device if there is no routes

**Comment by GoodiesHQ** [comment]
Agreed on both! I wanted to hide those nodes with no routes in the first place, I can't remember why I didn't. I'll add this functionality to the dev branch and let you know when it is available so you can use the :dev tag on Dockerhub and test it out. Thanks!

**Comment by GoodiesHQ** [comment]
![Image](https://github.com/user-attachments/assets/ac64dd4e-9e80-46ae-a3e3-3c6013285047)

I know this was a long delay, but now that I have fully migrated from Svelte 4 to Svelte 5, I am revisiting these types of features. I have added this to the dev branch at this point. It will be merged into main on the next update in the coming days. Thank you again for the recommendation and sorry it took so long!
